tensorwaves/physics/helicityformalism/amplitude.py

```diff
@@ -387,46 +386,84 @@ def _create_sequential_amplitude(
         "inv_mass_name",
         "inv_mass_name_prod1",
         "inv_mass_name_prod2",
+        "meson_radius",
```

Comment:
I think we should switch to [`typing.NamedTuple`](https://docs.python.org/3/library/typing.html#typing.NamedTuple) as it allows for much more powerful syntax than `collections.namedtuple`. Create an issue for this?

Note that we already enfoce Python >3.6, so that should be fine.

Reply:
Yeah, can you create an issue for that and we deal with that separately